core/embed/rust/src/ui/model_mercury/flow/confirm_action.rs

```diff
@@ -69,21 +69,21 @@ impl ConfirmActionStrings {
 
 #[derive(PartialEq)]
 pub struct ConfirmActionMenuStrings {
-    verb_cancel: Option<TString<'static>>,
+    verb_cancel: TString<'static>,
     has_info: bool,
     verb_info: Option<TString<'static>>,
 }
 
 impl ConfirmActionMenuStrings {
     pub fn new() -> Self {
         Self {
-            verb_cancel: None,
+            verb_cancel: TR::buttons__cancel.into(),
```

Comment:
maybe we can introduce:
```rust 
const VERB_CANCE_DEFAULT: TranslatedString = TR::buttions__cancel`;
```
and use it on the `unwrap_or` calls?

Reply:
Perfect! https://github.com/trezor/trezor-firmware/pull/4346/commits/7218e99421ca31fffbc85b5f4921b5e1afad2af8